http/server.ts

```diff
@@ -274,11 +274,15 @@ export class Server implements AsyncIterable<ServerRequest> {
     if (bufStateErr === "EOF") {
       // The connection was gracefully closed.
     } else if (bufStateErr instanceof Error) {
-      // An error was thrown while parsing request headers.
-      await writeResponse(req.w, {
-        status: 400,
-        body: new TextEncoder().encode(`${bufStateErr.message}\r\n\r\n`)
-      });
+      // in case of wrong first line HTTP header or
+      // aborted request we don't have to respond
+      if (req) {
```

Comment:
I think we still run the risk of responding to the *previous* request. 
This could be fixed by adding `req = undefined` to the start of the while loop above.

Reply:
TBH i don't know if this case could happen. As i haven't found a way to test it. I trust you on this.